site/ar_LB.all.json

```diff
@@ -1575,5 +1575,38 @@
   },
   "new_card_error_timed_out": {
     "other": "حدث خطأ ما عند إضافة تلك البطاقة. يرجى المحاولة مرة أخرى في وقت لاحق أو استخدام بطاقة مختلفة."
+  },
+  "sso_error_admin_requires_mfa": {
+    "other": "غير قادر على تسجيل الدخول. لأسباب أمنية ، تتطلب حسابات المسؤول مصادقة متعددة العوامل (MFA)."
```

Comment:
`(MFA)` is this a Google Translate fail?

Reply:
Unsure how we verify this, translate it back?

Reply:
Usually the hard way... Googling and common-sense best guesses.  In this case... I would probably just remove `(MFA)`, run the translator, then add `(MFA)` back into English since it's the only one I can be sure about.

Reply:
Thanks I have given this a go